ftpvl/fetchers.py

```diff
@@ -68,16 +74,88 @@ def __init__(
         project: str,
         jobset: str,
         eval_num: int = 0,
+        absolute_eval_num: bool = False,
         mapping: dict = None,
         hydra_clock_names: list = None
     ) -> None:
         super().__init__()
         self.project = project
         self.jobset = jobset
         self.eval_num = eval_num
+        self.absolute_eval_num = absolute_eval_num
         self.mapping = mapping
         self.hydra_clock_names = hydra_clock_names
 
+    def _get_builds(self, eval_num: int, absolute_eval_num: bool, params: str = "") -> List[int]:
+        """
+        Recursive function that returns a list of build numbers given an eval_num
+        and whether it is an absolute eval num.
+
+        Parameters
+        ----------
+        eval_num : int
+            An integer that specifies the evaluation to download. Functionality
+            differs depending on whether `absolute_eval_num` is True
+        absolute_eval_num : bool
+            Flag that specifies if the eval_num is an absolute identifier instead of
+            a relative identifier. If True, the fetcher will find an evaluation
+            with the exact ID in `eval_num`. If False, `eval_num` should be a
+            non-negative integer with `0` being the latest evaluation and `1` being
+            the second latest evaluation, etc.
+        params : str
+            A string of query parameters used when fetching the evaluations.
+            Most commonly used for pagination. By default, "".
+
+        Returns
+        -------
+        List[int]
+            A list of builds that correspond with the eval_num
+
+        Raises
+        ------
+        ConnectionError
+            Raised if the HTTP request to get the evaluations fails.
+        IndexError
+            Raised if the relative eval_num is not valid.
+        ValueError
+            Raised if the eval_num has no associated builds.
+        """
+        resp = requests.get(
+            f"https://hydra.vtr.tools/jobset/{self.project}/{self.jobset}/evals{params}",
+            headers={"Content-Type": "application/json"},
+        )
+        if resp.status_code != 200:
+            raise ConnectionError("Unable to get evals from server.")
+        evals_json = resp.json()
+        
+        if not self.absolute_eval_num: # if relative eval_num
```

Comment:
Since both cases of this if statement are about equally complex, it might read better to have the `self.absolute_eval_num` case first, then the relative one, as follows:
```
if self.absolute_eval_num:
  < absolute eval_num case code >
else:
  < relative eval_num case code >
```
Then you could skip the comment after the `if` (but still probably keep a comment after the `else`)